base_layer/core/src/chain_storage/postgres_db/migrations/2020-04-07-14-49_tx_outputs/up.sql

```diff
@@ -0,0 +1,16 @@
+create table tx_outputs(
```

Comment:
```suggestion
create table if not exists tx_outputs(
```

Please make all these migrations idempotent